Source/Core/BoundingSphere.js

```diff
@@ -1110,6 +1110,26 @@ define([
     };
 
     /**
+     * Computes the distance between a sphere and the camera attached to frameState.
+     * @param {BoundingSphere} [sphere] The bounding sphere.
+     * @param {FrameState} [frameState] The frameState to which the camera is attached.
+     * @returns {Number} The distance between the camera and the bounding sphere in meters. Returns 0 if the camera is inside the bounding volume.
+     *
+     * @private
+     */
+    BoundingSphere.distanceToCamera = function(sphere, frameState) {
```

Comment:
Since `BoundingSphere` is in `Core`, and `FrameState` is in `Scene`, this really shouldn't take a `frameState` parameter.  Perhaps we can get around this by introducing a small private `BoundingSphere` wrapper in `Scene` that adds this function.